ykman-gui/py/yubikey.py

```diff
@@ -34,6 +34,17 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+class PivContextManager:
```

Comment:
Since we decided offline that we want this, I'll try to prepare a PR that does something similar for OTP and FIDO.